packages/ipfs-cli/src/commands/dag/put.js

```diff
@@ -4,6 +4,8 @@ const mh = require('multihashing-async').multihash
 const multibase = require('multibase')
 const dagCBOR = require('ipld-dag-cbor')
 const dagPB = require('ipld-dag-pb')
+/** @type {typeof import('it-concat').default} */
+// @ts-ignore - incorrect typedefs
```

Comment:
Does this PR resolve the problem? https://github.com/alanshaw/it-concat/pull/4

Reply:
It does, but I did not wanted to block this. I would much rather land with comment and then address all the ts-ignores as followups.

Reply:
It's been merged so this should be ok to remove now?